dandiapi/api/tests/test_version.py

```diff
@@ -90,8 +91,9 @@ def test_published_version_metadata_computed(published_version: Version):
         **original_metadata,
         'manifestLocation': [
             (
-                f'http://localhost:9000/test-dandiapi-dandisets/test-prefix/dandisets'
-                f'/{published_version.dandiset.identifier}/{published_version.version}/assets.yaml'
+                f'http://{os.getenv("DJANGO_MINIO_STORAGE_ENDPOINT")}/test-dandiapi-dandisets'
```

Comment:
This should be available from settings:
```suggestion
                f'http://{settings.MINIO_STORAGE_ENDPOINT}/test-dandiapi-dandisets'
```